.github/workflows/e2e.yaml

```diff
@@ -37,6 +37,18 @@ on:
 jobs:
   e2e-tests:
     runs-on: linux-amd64-cpu4
+    strategy:
+      matrix:
+        ispr:
+          - ${{github.event_name == 'pull_request'}}
+        include:
+          - variant: EOL
+            holodeck_file: tests/e2e/infra/aws_EOL.yaml
+          - variant: LATEST
+            holodeck_file: tests/e2e/infra/aws_LATEST.yaml
+        exclude:
+          - variant: EOL
+            ispr: true
```

Comment:
I would rather run against an older driver on PRs and LATEST on main.

Reply:
I see, ok